WalletWasabi.Gui/WalletWasabi.Gui.csproj

```diff
@@ -56,11 +56,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Avalonia.Desktop" Version="0.9.4" />
-    <PackageReference Include="AvalonStudio.Shell" Version="0.9.4" />
-    <PackageReference Include="Dock.Avalonia" Version="0.9.4.1" />
-    <PackageReference Include="Dock.Avalonia.Themes.Default" Version="0.9.4.1" />
-    <PackageReference Include="Dock.Avalonia.Themes.Metro" Version="0.9.4.1" />
+    <PackageReference Include="Avalonia.Desktop" Version="0.9.5" />
+    <PackageReference Include="AvalonStudio.Shell" Version="0.9.5" />    
+    <PackageReference Include="Dock.Avalonia.Themes.Default" Version="0.9.5" />
```

Comment:
The `Dock.Avalonia` package ref seems to be missing?

Reply:
Its supposed to be, we had to add it back because we needed to overide Dock with hotfix... now the versions are aligned we dont need it.